lab-david/model/user.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
```

Comment:
I am a really big fan of writing a `User.create` method so that you can write cleaner routes. Also makes your mocks cleaner.